tests/testSed.py

```diff
@@ -139,12 +152,12 @@ def test_read_sed_flambda(self):
         Test how readSED_flambda handles the reading of SED filenames
         when we fail to correctly specify their gzipped state.
         """
-        scratch_dir = os.path.join(getPackageDir("sims_photUtils"),
-                                   "tests", "scratchSpace")
```

Comment:
I think using {{teardown_module}} is a path of last resort. I don't really like relying on it. If scratch_dir is only used in that one test, I think I'd prefer a simply mkdtemp/rmtree just in that test. In this particular case it looks like `getTempFilePath` would mostly work (except for the `assertRaises` test).